app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -357,7 +368,9 @@ class MainActivity : ComponentActivity() {
         ) {
 
             Row {
-                Text(text = "TV Tracker", fontSize = 35.sp, fontWeight = FontWeight.Bold, modifier = Modifier.padding(vertical = 40.dp))
+                Text(text = "TV Tracker", fontSize = 35.sp,
+                    fontWeight = FontWeight.Bold,
```

Comment:
Broke up UI properties for added readability to display different elements implementations, as well as to wrap lines that were too long.
https://developer.android.com/kotlin/style-guide#where_to_break